src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+// create interface 'Shape' & 'Color' with three classes implementing
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+// create function error message
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)
Fix all cases